onnxruntime_customops/_ortcustomops.pyi

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Is this file needed?

Reply:
Yes, it's very important for C++ implemented function and class on the type verification.